### DISCUSSION TITLE
Welcome to hassio-bluetti-bt Discussions!

### DISCUSSION THREAD

**Discussion by Patrick762** [Patrick762/hassio-bluetti-bt#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Please not that I only look into discussions very rarely, so it's more a place to communicate with others using this integration.

**Comment by andrewagner86** [comment]
Hi @Patrick762 at first, I really like your HACS integration. Is there a slack channel or something else to contact you? I have an ep600 and would like to help you with the integration.

**Reply by Patrick762** [reply]
Currently there's no Slack Channel. Maybe I'll start a channel. But more likely on discord

**Reply by andrewagner86** [reply]
Discord is also ok for me. :)

**Reply by Patrick762** [reply]
Here's a link to the new discord server: https://discord.gg/nqcgV2MW

**Reply by Fridgeir-9** [reply]
The invitation has expired. Please provide a new one

**Reply by Patrick762** [reply]
https://discord.gg/zc8BKjbb

**Comment by tantox1** [comment]
Hi, I'm Alessio. I've just bought an AC70 and I'm wondering how to proper use the integration. I'll open a discussion. in the mean, I want to say thank you to the creator of the integration for his work

**Comment by jakeborja** [comment]
This was a game changer for our off grid set up using the AC500. Is there any way to also get error messages?

**Comment by krikey** [comment]
I'm setting up a RPi3 in my van with hassio-bluetti-bt to connect to an AC180.

For some reason when I configure the plugin, it doesn't list the Bluetti device name, but lists what look like Bluetooth addresses, none of which see to be the Bluetti AC180.

**Reply by GoensGoens** [reply]
Hello krikey, I just startet with Patricks software and I have all ready and can see a lot different mac-adresses and I do not know, which one is my AC500. Did you solve this problem ?

**Reply by krikey** [reply]
No I didn't solve the issue and my AC180 went back after it failed to charge after 3 weeks of light use.

**Comment by user150aa346** [comment]
Patrick, thanks for the HACS extension. I have an AC200L and it's I'm getting more info than through Bluetti's crappy ios app. I did have a question. Is there a way to display "time remaining"? On the display of the AC200L it shows how many hours are left. It would nice to be able to display that on my dashboard. Thanks. Mike

**Comment by LTELIS** [comment]
hello my friend,
Please add support for AC50B.
